Neos.Neos/Classes/Aspects/NodeTypeConfigurationEnrichmentAspect.php

```diff
@@ -161,6 +167,49 @@ protected function resolveHelpMessageThumbnail($nodeTypeName, $configurationThum
         }
     }
 
+    /**
+     * @param string $nodeTypeLabelIdPrefix
+     * @param string $propertyName
+     * @param array $propertyConfiguration
+     * @param bool $isCreationDialogElement
+     * @return void
+     */
+    protected function applyValidationLabels($nodeTypeLabelIdPrefix, $propertyName, array &$propertyConfiguration, $isCreationDialogElement = false): void
+    {
+        $hasValidator = isset($propertyConfiguration['validation']);
+        if ($hasValidator && is_array($propertyConfiguration['validation'])) {
+            foreach ($propertyConfiguration['validation'] as $validatorName => $validatorOptions) {
+                if ($this->shouldFetchTranslation($validatorOptions, 'validationErrorMessage')) {
+                    $validatorNamePieces = explode('/', $validatorName);
+                    $validatorNameForLabel = lcFirst(array_pop($validatorNamePieces));
+                    $path = 'validation.' . $validatorNameForLabel . '.validationErrorMessage';
+                    $propertyConfiguration['validation'][$validatorName]['validationErrorMessage'] = $this->getValidationElementConfigurationTranslationId(
+                        $nodeTypeLabelIdPrefix,
+                        $propertyName,
+                        $path,
+                        $isCreationDialogElement
+                    );
+                }
+            }
+        }
+    }
+
+    /**
+     * Returns translation ID for the validation configuration.
+     *
+     * @param string $nodeTypeLabelIdPrefix
+     * @param string $propertyName
+     * @param string $path
+     * @param bool $isCreationDialogElement
+     * @return string
+     */
+    protected function getValidationElementConfigurationTranslationId(string $nodeTypeLabelIdPrefix, string $propertyName, string $path, bool $isCreationDialogElement): string
+    {
+        return $isCreationDialogElement ?
+            $this->getCreationDialogElementsConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, $path) :
+            $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, $path);
```

Comment:
```suggestion
        return $isCreationDialogElement
            ? $this->getCreationDialogElementsConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, $path)
            : $this->getPropertyConfigurationTranslationId($nodeTypeLabelIdPrefix, $propertyName, $path);
```
i was just wondering if this looks better ... couldnt find anything in PSR2

Reply:
Maybe it is more my style that is influenced by the JS world.

https://eslint.org/docs/rules/multiline-ternary

Reply:
Is this ok, or should we discuss this with a wider audience?

Reply:
not that big of a deal should not block your pr in any way! - just kept this open, as i thought someone could give me a lead of whats the php way of doing it ^^ - since im still relatively new to php